src/llama_stack_client/resources/inference.py

```diff
@@ -408,6 +409,11 @@ def chat_completion(
         extra_body: Body | None = None,
         timeout: float | httpx.Timeout | None | NotGiven = NOT_GIVEN,
     ) -> ChatCompletionResponse | Stream[ChatCompletionResponseStreamChunk]:
+        warnings.warn(
```

Comment:
hm, this is actually auto-generated code. I think we'd need to look into stainless to see if this can be done.